DataProcessing/datax-host/pom.xml

```diff
@@ -118,13 +118,13 @@ SOFTWARE
         <dependency>
             <groupId>com.microsoft.datax</groupId>
             <artifactId>datax-core_2.4_2.11</artifactId>
-            <version>1.2.3-SNAPSHOT</version>
+            <version>1.2.4-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>com.microsoft.datax</groupId>
             <artifactId>datax-utility_2.4_2.11</artifactId>
             <version>1.2.3-SNAPSHOT</version>
```

Comment:
The version of all the datax dependency packages would change to 1.2.4-SNAPSHOT

Reply:
Fixed